src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/services/CamundaServiceTest.java

```diff
@@ -2040,7 +2039,7 @@ void should_auto_complete_and_return_empty_array() {
         }
 
```

Comment:
can we add a unit to also cover wrong jurisdictionType?

Reply:
I think this "searchWithCriteria_should_return_an_empty_task_list_when_camunda_takes_wrong_caseType" covers for the wrong jurisdictionType. 
Do you agree?